angular.json

```diff
@@ -210,6 +384,28 @@
             }
           }
         },
+        "server-cds": {
+          "builder": "@angular-devkit/build-angular:server",
+          "options": {
+            "outputPath": "dist/storefrontapp-server",
+            "main": "projects/storefrontapp/server.ts",
+            "tsConfig": "projects/storefrontapp/tsconfig.server.json"
+          },
+          "configurations": {
+            "production": {
+              "outputHashing": "media",
+              "fileReplacements": [
+                {
+                  "replace": "projects/storefrontapp/src/environments/environment.ts",
+                  "with": "projects/storefrontapp/src/environments/cds/cds.environment.prod.ts"
```

Comment:
You should replace environment.ts with environment.prod.ts to have idential behavior with normal production build and the second replacement for cds.environment.ts should be to cds.environment.dev.ts. We don't have production configuration.
Additionally inclue occ-environment to be replaced with 1905 version like in normal production build (also applies to normal server)

Reply:
Thanks for pointing it out, done.